tests/TitleTest.php

```diff
@@ -1339,7 +1350,7 @@ public function testGoofs()
         $goofs = $imdb->goofs();
         $this->assertInternalType('array', $goofs);
         $this->assertGreaterThan(135, count($goofs));
-        $this->assertLessThan(145, count($goofs));
+        $this->assertLessThan(146, count($goofs));
```

Comment:
Merge conflict because you've changed this test and I've changed it in master. Could you revert your change or take this change to match master
```suggestion
        $this->assertLessThan(150, count($goofs));
```